configuration/pipeline_config.json

```
@@ -116,7 +116,16 @@
   "AutomatedAnalysis": {
     "GenerateRegionThemeDistributionMaps": true,
     "GenerateDistrictThemeDistributionMaps": true,
-    "GenerateMogadishuThemeDistributionMaps": true
+    "GenerateMogadishuThemeDistributionMaps": true,
+    "TrafficLabels": [
+      {"StartDate": "2021-09-10T00:00+03:00", "EndDate": "2021-09-12T23:00+03:00", "Label": "E01 Promo"},
```

```suggestion
      {"StartDate": "2021-09-10T00:00+03:00", "EndDate": "2021-09-13T16:30+03:00", "Label": "E01 Promo"},
```
This should cover upto sms ad time.